src/main/java/org/candlepin/resteasy/filter/StoreFactory.java

```diff
@@ -59,14 +60,29 @@ public class StoreFactory {
     @Inject
     public StoreFactory(Injector injector) {
         storeMap.put(Owner.class, new OwnerStore());
-        storeMap.put(Environment.class, new EnvironmentStore());
-        storeMap.put(Consumer.class, new ConsumerStore());
-        storeMap.put(Entitlement.class, new EntitlementStore());
-        storeMap.put(Pool.class, new PoolStore());
+        storeMap.put(Environment.class, new EnvironmentStore(
+            injector.getInstance(EnvironmentCurator.class)));
+        storeMap.put(Consumer.class, new ConsumerStore(
+            injector.getInstance(ConsumerCurator.class),
+            injector.getInstance(DeletedConsumerCurator.class),
+            injector.getInstance(I18nProvider.class),
+            injector.getInstance(OwnerCurator.class)));
+        storeMap.put(Entitlement.class, new EntitlementStore(
+            injector.getInstance(EntitlementCurator.class)
+        ));
+        storeMap.put(Pool.class, new PoolStore(
+            injector.getInstance(PoolCurator.class)
+        ));
         storeMap.put(User.class, new UserStore());
-        storeMap.put(ActivationKey.class, new ActivationKeyStore());
-        storeMap.put(Product.class, new ProductStore());
-        storeMap.put(AsyncJobStatus.class, new AsyncJobStatusStore());
+        storeMap.put(ActivationKey.class, new ActivationKeyStore(
+            injector.getInstance(ActivationKeyCurator.class)
+        ));
+        storeMap.put(Product.class, new ProductStore(
+            injector.getInstance(ProductCurator.class)
+        ));
+        storeMap.put(AsyncJobStatus.class, new AsyncJobStatusStore(
+            injector.getInstance(AsyncJobStatusCurator.class)
+        ));
 
         for (EntityStore<? extends Persisted> store : storeMap.values()) {
             injector.injectMembers(store);
```

Comment:
This is no longer necessary (or recommended) if we're removing property injection.